### PR TITLE
Breadcrumb refactor dark

### PR DESCRIPTION
**Pull request by wellhairy**

### CODE REVIEW

**Comment by natmunckton** [comment]
Happy to merge this now

File: src/scss/core/structure/_breadcrumb.scss [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        display: block;
+      }
+
+      :last-of-type #{$this}__arrow {
+        margin-right: 0;
+      }
+    }
+  }
+
+  &__show-all-button ,
```
Comment (LidorLapid) [inline src/scss/core/structure/_breadcrumb.scss]: Extraneous space and indentation issue here.

File: src/scss/core/structure/_breadcrumb.scss [hunk 2]
```diff
@@ -0,0 +1,68 @@
+.breadcrumb {
+  $this: &;
+
+  margin-top: -8px;
+  padding-top: 5px;
```
Comment (LidorLapid) [inline src/scss/core/structure/_breadcrumb.scss]: Padding properties can be combined into shorthand.

File: gulpfile.js [hunk 3]
```diff
@@ -89,7 +89,7 @@ gulp.task('watch', () => {
   gulp.watch([`${paths.src.scss}/**/*.scss`], ['build-scss']);
   gulp.watch([`${paths.src.imgs}/**/*`], ['minify-images']);
   gulp.watch([`${paths.src.icons}/**/*`], ['build-sprites']);
-  gulp.watch([`${paths.src.svgs}/**/*`], ['minify-svgs']);
+  gulp.watch([`${paths.src.svgs}/**/*`], ['minify-svgs', 'build-jekyll']);
```
Comment (LidorLapid) [inline gulpfile.js]: I think this would be a bit cleaner if we added the SVGs path to our existing build-jekyll watch command.

Comment (wellhairy) [reply gulpfile.js]: Yeah I have no problem with that. Just a thought though. do we need this to happen sequentially anyway? if so we might want to thin about putting it in a run sequence layer.

**Pull request merged**